cmake/package.cmake

```diff
@@ -5,6 +5,7 @@ set_target_properties(
 install(TARGETS
   onnxruntime-genai
   LIBRARY DESTINATION lib
+  RUNTIME DESTINATION lib
```

Comment:
Does this change have any impact on Linux?

Reply:
No, we don't currently have any executable.